### PR TITLE
edit documentation

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: wiki/Manual.md [hunk 1]
```diff
@@ -3474,7 +3474,7 @@ We present a unified structure for the positions returned by exchanges.
    'hedged': false,             // boolean, whether or not the position is hedged, i.e. if trading in the opposite direction will close this position or make a new one
    'side': 'long',              // string, long or short
    'contracts': 5,              // float, number of contracts bought, aka the amount or size of the position
-   'contractSize': 100,         // float, the size of one contract in quote units
+   'contractSize': '100',       // string, the size of one contract in quote units
```
Comment (xmatthias) [inline wiki/Manual.md:3477]: I don't think this change is correct.

The documentation section is about the "position structure" -> the position return value - which seems to be a number:
https://github.com/ccxt/ccxt/blob/master/js/ftx.js#L1962
https://github.com/ccxt/ccxt/blob/master/js/gateio.js#L2954

Comment (frosty00) [reply wiki/Manual.md:3477]: @xmatthias you're right

File: wiki/Manual.md [hunk 2]
```diff
@@ -729,32 +729,45 @@ Each currency is an associative array (aka dictionary) with the following keys:
...
+    'quote':   'USD',         // uppercase string, unified quote currency code, 3 or more letters
+    'baseId':  'btc',         // any string, exchange-specific base currency id
+    'quoteId': 'usd',         // any string, exchange-specific quote currency id
+    'active':   true,         // boolean, market status
+    'taker':    0.002,        // taker fee rate, 0.002 = 0.2%
+    'maker':    0.0016,       // maker fee rate, 0.0016 = 0.16%
+    'type':    'spot'         // spot for spot, future for expiry futures, and swap for perpetual swaps
+    'linear':   true          // the contract is a linear contract (settled in quote currency)
+    'inverse':  false         // the contract is an inverse contract (settled in base currency)
+    'contractSize': '1'       // the size of one contract, only used if `contract` is true
```
Comment (xmatthias) [inline wiki/Manual.md:744]: While it's good to document this - i do feel that this is a huge missalignment within ccxt - and a huge bummer to usability (as such, this PR will not fix #11123).

We have the same exact naming (`'contractSize'`) in different endpoints, basically requiring users to "know" which endpoint returns which format (this is something that i'm yet to find any documentation on. I can guess this for most endpoints - but the documentation does not include any hint at "endpoint X returns the object Y".


As such, i consider this as a bug in ccxt (same attribute returns different values for different endpoints), which is not just a documentation problem.

Comment (xmatthias) [inline wiki/Manual.md:744]: While it's good to document this - i do feel that this is a huge missalignment within ccxt - and a huge bummer to usability (as such, this PR will not fix #11123).

We have the same exact naming (`'contractSize'`) in different endpoints, basically requiring users to "know" which endpoint returns which format (this is something that i'm yet to find any documentation on. I can guess this for most endpoints - but the documentation does not include any hint at "endpoint X returns the object Y".


As such, i consider this as a bug in ccxt (same attribute returns different values for different endpoints), which is not just a documentation problem.

**Review by xmatthias — CHANGES_REQUESTED** [review]

File: wiki/Manual.md (hunk 2 above)
Comment (frosty00) [reply wiki/Manual.md:744]: > As such, i consider this as a bug in ccxt (same attribute returns different values for different endpoints), which is not just a documentation problem.

We're storing the data as a string and returning it as a number for convenience in `fetchPositions`, we need to store it as a string for precision with our string math functions

Comment (xmatthias) [reply wiki/Manual.md:744]: You could store it in a property like `'contractSize_str'` for convenience within ccxt - but for the user, an aligned type for the same method should be desired.

**Pull request merged**